include/nanobind/stl/bind_map.h

```diff
@@ -0,0 +1,196 @@
+/*
+    nanobind/stl/bind_map.h: nb::bind_map()
```

Comment:
`nb::bind_map()` -> `Automatic creation of bindings for map-style containers`

Reply:
Added in 312a763.